Makefile

```diff
@@ -102,6 +112,101 @@ init:
 		-v "`pwd`/test/share/letsencrypt":/etc/letsencrypt \
 		-t $(NAME)
 
+	docker run \
+		-d \
+		--name mailserver_ldap \
+		--link openldap \
+		--link redis:redis \
+		-e LDAP_ENABLED=true \
+		-e LDAP_HOST=openldap \
+		-e LDAP_BIND_DN="cn=admin,dc=domain,dc=tld" \
+		-e LDAP_BIND_PW="testpasswd" \
+		-e LDAP_DEFAULT_SEARCH_BASE="o=mx,dc=domain,dc=tld" \
+		-e LDAP_DOMAIN_FILTER="(&(mail=*@%s)(objectClass=mailAccount))" \
+		-e LDAP_DOMAIN_ATTRIBUTE="mail" \
+		-e LDAP_DOMAIN_FORMAT="%d" \
+		-e LDAP_MAILBOX_FILTER="(&(mail=%s)(objectClass=mailAccount))" \
+		-e LDAP_MAILBOX_ATTRIBUTE="mail" \
+		-e LDAP_MAILBOX_FORMAT="/var/mail/vhosts/%d/%s/mail/" \
```

Comment:
LDAP_MAILBOX_FORMAT is the biggest confusing part during my deployment. It seems that there is some inconsistency on how mailbox path is composed between different places:

1. ldap vs sql in postfix
In /etc/postfix/main.cf, the base path of the mailbox path is already set and according to [postfix document](http://www.postfix.org/postconf.5.html#virtual_mailbox_maps), it seems that it is always prepended to the path from lookup. 
```
virtual_mailbox_base    = /var/mail
```
and in the sql part, looking at its tests, the maildir field in sql table is stored like `domain.tld/john.doe/`.

Therefore, it seems that LDAP_MAILBOX_FORMAT is better to be set without the prefix?

2. postfix vs dovecot
It seems that these two configurations for postfix and dovecot are not compatible:
```
LDAP_MAILBOX_FORMAT="/var/mail/vhosts/%d/%s/mail/"
LDAP_DOVECOT_USER_ATTRS="=home=/var/mail/vhosts/%d/%n/,=mail=maildir:/var/mail/vhosts/%d/%n/mail/...
```

I am very new to both postfix and dovecot, but from their documents, it seems that %s in postfix represents the entire email address and %n in dovecot represents only the username part.

To be consistent with sql and dovecot, should `LDAP_MAILBOX_FORMAT="%d/%u/"` be recommended?

On the other hand, since in /etc/postfix/main.cf, lmtp transport to dovecot is used. Does it mean that LDAP_MAILBOX_FORMAT value is actually not used?
```
virtual_transport       = lmtp:unix:private/dovecot-lmtp
```